src/utils/validations/presence/presence.js

```diff
@@ -76,21 +76,24 @@ class PresenceValidator {
    * @return {Boolean} true if value is valid
    */
   validate = (value, props) => {
-    let valid, result = this.requireAll;
+    let valid,
+        result = this.requireAll,
+        val = value;
 
     if (!this.props) {
       return isValid(value);
-    } else {
-      forEach(this.props, (name) => {
-        value = props[name];
-        valid = isValid(value);
-        result = this.requireAll ? (result && valid) : (result || valid);
-        if (result !== this.requireAll) {
-          return false;
-        }
-      });
-      return result;
     }
+
+    forEach(this.props, (name) => {
+      val = props[name];
+      valid = isValid(val);
+      result = this.requireAll ? (result && valid) : (result || valid);
+      if (result !== this.requireAll) {
+        return false;
+      }
+      return true;
```

Comment:
I don't believe that is the case in JavaScript. I think it will return from the function. Its using lodash which doesn't add much docs - https://lodash.com/docs/4.17.4#forEach